### PR TITLE
documentation how to use gateway.

### PR DESCRIPTION
**Pull request by PawelPawelec-RDX**

### CODE REVIEW

File: src/RadixDlt.NetworkGateway.GatewayApi/examples.md [hunk 1]
```diff
@@ -0,0 +1,446 @@
+# How to use Gateway
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/examples.md:1]: I left it as seprate file but we have to move it to gateway-api-schema.yaml under examples section.

File: src/RadixDlt.NetworkGateway.GatewayApi/examples.md [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+```
+{ 
+  ...
+  "order": "Asc" 
+  ...
+}
+```
+Keep in mind that if order is ascending you'll start fetching transactions from oldest first and with each page you'll fetch newer transactions.
+
+### TODO PP: parsing events to determine what happened in a particular dApp
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/examples.md:425]: We have to fill it in.

File: src/RadixDlt.NetworkGateway.GatewayApi/examples.md [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+
+
+## Querying/parsing for resource behaviours
+
+### TODO PP: I'm not sure what exactly we should put here
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/examples.md:430]: We have to fill it in. I wasn't sure what to put here.

File: src/RadixDlt.NetworkGateway.GatewayApi/examples.md [hunk 4]
```diff
@@ -0,0 +1,446 @@
...
+
+## Querying for key value store entries (eg how to construct key to query with)
+
+`/state/key-value-store/data` Endpoint allows to get data stored under specific key in key value store.
+
+It takes `key_value_store_address` and `keys` array as parameter. Where `key_value_store_address` is simple entity address, `keys` items might be either in hex or json format.
+
+### Hex keys
+#### TODO PP: fill it in
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/examples.md:439]: We have to fill it in.

File: src/RadixDlt.NetworkGateway.GatewayApi/examples.md [hunk 5]
```diff
@@ -0,0 +1,446 @@
...
+
+### Json keys
+#### TODO PP: fill it in
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/examples.md:442]: We have to fill it in.

File: src/RadixDlt.NetworkGateway.GatewayApi/examples.md [hunk 6]
```diff
@@ -0,0 +1,446 @@
...
+
+
+## Imagine a component with two XRD vaults - parse state and determine value of each vault
+### TODO PP: fill it in
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/examples.md:446]: We have to fill it in.

File: src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml [hunk 7]
```diff
@@ -101,6 +102,11 @@ tags:
       - `next_cursor` *(optional)* if present indicates that the next chunk (page) exists and can be fetched using collection-specific endpoint and cursor value,
       - `items` a chunk (page) of unerlying collection items.
   - name: Concepts
+  - name: Examples
+    x-displayName: Examples
+    x-traitTag: true
+    description: |
+        TODO FILL IT IN.
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml:109]: move entire examples.md here so it'll be automatically rendered in redocly as other secitons.

File: src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml [hunk 8]
```diff
@@ -101,6 +102,689 @@ tags:
...
+{ 
+  ...
+  "order": "Asc" 
+  ...
+}
+```
+Keep in mind that if order is ascending you'll start fetching transactions from oldest first and with each page you'll fetch newer transactions.
+
+### Reading events to find out what happened in certain dApp.
+### TODO PP:
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml:535]: we support only withdraw/deposit event. I could show how to filter using `affected_global_entities` filter here but it's based on substates so not sure if that's what we need here.

**Review by dhedey — APPROVED** [review]

**Pull request merged**